components/browser/state/src/main/java/mozilla/components/browser/state/state/CustomTabSessionState.kt

```diff
@@ -19,7 +19,8 @@ data class CustomTabSessionState(
     override val content: ContentState,
     override val trackingProtection: TrackingProtectionState = TrackingProtectionState(),
     val config: CustomTabConfig,
-    override val engineState: EngineState = EngineState()
+    override val engineState: EngineState = EngineState(),
+    override val extensionState: List<WebExtensionState> = emptyList()
```

Comment:
I feel like the global and the tab `WebExtensionState` should actually be two separate objects. It's not unlikely that we will have more global state properties that will not make sense on the tab-local state. What do you think?

Reply:
> It's not unlikely that we will have more global state properties that will not make sense on the tab-local state. What do you think?

To be sure, did you mean  with **It's not unlikely** == **it likely**? :)
Do you have any examples on mind?

Reply:
Let's approach this later as right now we don't have all the details about which properties only make sense on tab-local vs global one.